automation/deploy_igz/deploy_env_template.yml

```
@@ -0,0 +1,35 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The deploy scripts expect the yaml file to be in the same directory
+# this is a template file for the deploy env file
+
+
+# Data node IPs/hostnames (required)
+DATA_NODES:
+
+# Username for ssh (required)
+USER:
+
+# Password for above user (required)
+PASSWORD:
+
+# Docker registry you can push to. Make sure you are logged into it before running the script
+DOCKER_REGISTRY:
```

I suggest to add this template:
```suggestion
DOCKER_REGISTRY: docker.io/<username>
```